phys2bids/phys2bids.py

```diff
@@ -380,6 +380,16 @@ def phys2bids(filename, info=False, indir='.', outdir='.', heur_file=None,
                 phys_out[key].timeseries.insert(0, np.linspace(phys_in[run].timeseries[0][0],
                                                 phys_in[run].timeseries[0][-1],
                                                 num=phys_out[key].timeseries[0].shape[0]))
+            # add trigger channel in the proper frequency
```

Comment:
```suggestion
            # Add trigger channel in the proper frequency.
```

Reply:
I think this changed when merging with master, sry about it